File: TunnelProject/Opc/Enums.cs
namespace TunnelProject.Opc
{
    public enum OPC_COMM_TYPE : int
    {
        OpcCommStatus = 1,
        OPCTagValueChanged = 2
    }

    public enum OpcThreadMode : int
    {
        MULTI_THREADED,
        SINGLE_THREADED
    }

    public enum CipMedia : sbyte
    {
        Ethernet = 18,
        Backplane = 1
    }
}

File: TSA_Rail_System_TunnelProject/Wdw_Sust.OpcDa.Core/Enums.vb
Public Module Enums

    Public Enum CipMedia As SByte
        Ethernet = 18
        Backplane = 1
    End Enum

    Public Enum OPC_COMM_TYPE As Integer
        OpcCommStatus = 1
        OPCTagValueChanged = 2
    End Enum

    Public Enum OpcThreadMode As Integer
        MULTI_THREADED
        SINGLE_THREADED
    End Enum

End Module